tests/test_apigateway/test_apigateway.py

```diff
@@ -543,7 +543,9 @@ def test_integration_response():
         statusCode="200",
         selectionPattern="foobar",
         responseTemplates={},
+        contentHandling="CONVERT_TO_BINARY"
```

Comment:
Can you assert that the param can be retrieved as well? (In one of the tests - doesn't have to be this particular one.)

Also, please remove the contentHandling-param for some of the tests, to verify it's not mandatory.